### DISCUSSION TITLE
VPN Gateway do West Europe - packet loss

### DISCUSSION THREAD

**Discussion by adamhatak** [AzureCommunityPL/group#159] [Pomoc]
Koledzy, 
Czy obserwujecie w dni robocze w godzinach około 09:00 - 12:00 straty na pakietach do DC w West Europe? 
Mam zestawiony VPN Gateway (plan basic / legacy) i każdego dnia rano mam straty na pakietach na poziomie 5 - 15%.
Przełączałem tunel na plan Standard, ale efekt był taki sam.
Sprawdzałem po swojej stronie na dwóch różnych łączach internetowych (#1 50/50 Mbit/s, #2 500/500 Mbit/s).
Sprawdziłem u innego klienta (na innym łączu, na planie VpnGw1) i podobne problemy. 
Najbardziej odczuwalne problemy są na VoIP, gdzie centrala jest VMką w Azure. 
Czy obserwujecie coś podobnego u siebie?
Załączam kilka przykładowych analiz. Schemat analizy jest następujący:
- traceroute do IP VPN Gateway
- ping do najdalszego osiągalnego punktu przed siecią Microsoft lub zaraz na początku sieci Microsoft (np. 212.2.102.197)
- ping do najdalszego osiągalnego punktu przed VPN Gateway  (np. 104.44.21.184)
Do tego prosty ping z serwera on-prem do maszyny w Azure.

![image023](https://user-images.githubusercontent.com/70133147/100156879-fdf56280-2ea9-11eb-99ed-be8c916da0c8.png)
![image018](https://user-images.githubusercontent.com/70133147/100156883-ffbf2600-2ea9-11eb-8900-b709f563c2ea.png)
![image019](https://user-images.githubusercontent.com/70133147/100156882-ff268f80-2ea9-11eb-9867-85ee73bf6b09.jpg)

Testy wykonywane na dwóch różnych łączach. Mniej więcej po godzinie 12:00 wszystko zaczyna działać normalnie. Czas na pingach wraca do poziomu ~ 25ms. W czasie problemów średni czas to ~ 35-37 ms. To dane na łącza #1.

Pytanie czy obserwujecie u siebie takie problemy? Nie wiem od kiedy one występują, ale my zauważyliśmy je zaraz po migracji centrali VoIP do Azure i problemach z jakością połączeń.

Pozdrawiam.

**Comment by kaluzaaa** [comment]
Zdalne nauczanie i poranne callle na teams i innych.

**Reply by adamhatak** [reply]
A jest sposób na rozwiązanie problemu poza ExpressRoute? 🧐 North Europe?

**Reply by mgrabarz** [reply]
Chyba tylko express route zrobi robotę, albo przeczekaj te 3 tygodnie do świąt ;)))

**Reply by adamhatak** [reply]
@kaluzaaa @mgrabarz Dzięki za (p)odpowiedzi. 
A co po świętach? Ferie... Ale to tylko odwlekanie. Słabo to wygląda. Polska robi taki ruch, że DC West Europe siada?
Z supportu otrzymałem rekomendację aby przełączyć VPN Gateway na High performance SKU. Efekt taki sam. 
Jeżeli się nie mylę, to problem jest na trasie Bermin - Hamburg - Amsterdam. Tam się coś zapycha.

**Reply by MariuszFerdyn** [reply]
Sprawdz MTR gdzie gina te pakiety - swojego czasu ustalilem tak wlaniety router w OVH - ludzie na internecie tez o tym pisali. Niestety nie bylo jak dotrzec do wlasciciela routera... wiec trzeba bylo zmienic dostawce.

**Reply by senionit** [reply]
@adamhatak  Szybkie pytanie, czy ci dwaj providerzy nie idą tą samą ścieżką?

U mnie wygląda to tak:
```

Tracing route to ae140-0.icr03.ams06.ntwk.msn.net [104.44.21.184]
over a maximum of 30 hops:

  1     2 ms     1 ms     1 ms  192.168.28.1
  2    13 ms     4 ms    13 ms  100.108.64.1
  3     6 ms     6 ms     7 ms  185-2-37-8.internetia.net.pl [185.2.37.8]
  4    15 ms    16 ms    14 ms  warsc001rt06.inetia.pl [83.238.248.158]
  5    16 ms    16 ms    17 ms  ae64-0.waw01-96cbe-1a.ntwk.msn.net [104.44.12.12
]
  6    18 ms    18 ms    22 ms  ae20-0.icr02.mel20.ntwk.msn.net [104.44.239.10]

  7     *        *        *     Request timed out.
  8     *        *        *     Request timed out.
  9     *        *     ^C
```

Jak widać internetia wchodzi od razu do MSNa w warszawie, choć to może być bardzo indywidualne.


Jako, że wątek jest dość stary daj znać czy w ostatnim okresie się uspokoiło?

**Reply by adamhatak** [reply]
@senionit Dzięki za odpowiedź. Faktycznie w ostatnim czasie uspokoiło się. Nie obserwuję teraz strat na pakietach. Ale idę inną trasą. Miałem rozmowę z Microsoftem w tej sprawie i potwierdzali, że urządzenia brzegowe (HUB Warszawa) w godzinach mniej więcej 09:00 - 14:00 mają większe obciążenie, ale nie powinno to skutkować stratami na pakietach.
Z początkiem grudnia sytuacja się ustabilizowała, ale nie wiem czy to tylko dlatego, że ruch może być mniejszy, czy może faktycznie coś zrobili (inny trasa na traceroute).

**Reply by smereczynski** [reply]
Zobacz sobie to: https://docs.microsoft.com/en-us/azure/peering-service/about

**Comment by MariuszFerdyn** [comment]
Sprawdź MTR/ traceroute - czy po drodze jakiś router nie nawalił... niby ICMP, które może być różnie traktowane przez operatorów (niski priorytet) - ale pozwoliło to swojego czasu zdiagnozować i znaleźć strzelony router w OVH. A ileż to trzeba było pisać do OVH, aby coś z tym zrobili... 

Przy okazji ostatnie dwa moje wpisy w podobnym temacie:
https://rzetelnekursy.pl/easy-tool-for-checking-bandwidth-not-only-to-cloud-like-azure-aws-and-or-gcp/
https://rzetelnekursy.pl/how-to-optimize-your-vpn-connection-to-azure-gcp-and-aws/
